synapse/lib/ast.py

```diff
@@ -945,6 +945,9 @@ async def genx():
                 if runtsafe or not empty:
                     async for item in scmd.execStormCmd(runt, genr):
                         yield item
+            except s_exc.SynErr as e:
+                e.setdefault('stormcmd', name)
```

Comment:
we should probably talk about the name here

Reply:
We probably also need to discuss nesting/recursion here too...